### DISCUSSION TITLE
How to configure the service to find the user folders with init.d?

### DISCUSSION THREAD

**Discussion by transpetaflops** [abraunegg/onedrive#1347] [Q&A] [answered]
I apologize if this has been answered but I couldn't find it in the documentation. When starting onedrive in monitor mode with init.d, running as user root, how do I tell the service which user folders to monitor and what OneDrive accounts to use? I can see that the configuration for running it with systemd requires you to add the usernames to the service but I find no similar documentation for init.d

**Comment by abraunegg** [comment, marked as answer]
@transpetaflops 
To change the settings for an init.d setup, refer to https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#onedrive-service-running-as-root-user-via-initd

Once the correct user is setup, configure the `~/.config/<config-folder>/config` file with any additional application settings as required.

**Reply by transpetaflops** [reply]
Thank you for the reply. For this to work, is it a requirement to run the daemon as the user whose OneDrive I want to sync? What if I have multiple users on the system that each have their own OneDrive account? Currently I have one user on the system with its own config file in ~/.config/onedrive/config and the daemon is running as root and it does not see the changes made to the user's files.

**Reply by abraunegg** [reply]
Correct - you *have* to run the application as the user whose files you want to sync
